package/itest/ubuntu.sh

```diff
@@ -29,7 +29,8 @@ export TZ=US/Pacific
 ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 apt-get update && apt-get install -y software-properties-common
-apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli
+add-apt-repository ppa:deadsnakes/ppa
+apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli python3.7-distutils libatlas3-base
```

Comment:
not really sure we i had to add distutils here - afaik, this should have been installed as part of my debian/{rules,control} change

(the rest of these should also probably be in debian/control too so that they'd be automatically installed, probably)

Reply:
agree

Reply:
Agreed, yeah putting these as proper package dependencies would be much better if they are all truly needed, although things like `awscli` vs `aws-cli` might make that a bit more annoying

Reply:
yea, imma try to mess around with this tomorrow - i was too tired of messing with this today to do something less disgusting :p

Reply:
Stumbled over the same issue when I doing hacheck stuff, the paasta CLI in that container didnt work cause of missing distutils even though that should be installed via paasta's debian/control, probably fine to solve it the same way here

Reply:
not sure why we were installing awscli actually since we don't seem to actually use the cli in clusterman - i'm gonna go ahead and remove that

i think i also figured out why i had to re-add distutils here - i never referenced the `bionicOrLater:Depends` in debian/control

Reply:
err, my eyes are bad - we do use the aws cli

Reply:
nice, i think i got this working the right way now.